pkg/apis/experimental/v1alpha1/types.go

```diff
@@ -29,6 +29,10 @@ const (
 	// request whether the certificate should be marked as CA.
 	CertificateSigningRequestIsCAAnnotationKey = "experimental.cert-manager.io/request-is-ca"
 
+	// CertificateSigningRequestMaxPathLenAnnotationKey is the annotation key used to
+	// request a specific MaxPathLen for your CA certificate.
+	CertificateSigningRequestMaxPathLenAnnotationKey = "experimental.cert-manager.io/request-max-path-len"
```

Comment:
This is experimental because CertificateSigningRequest support is experimental, the IsCA annotation key is also experimental: https://github.com/cert-manager/cert-manager/blob/7f559528cd81ca5069e6900dd15f3eea76bd8f8f/pkg/apis/experimental/v1alpha1/types.go#L30

Reply:
Ahh, I missed that - thanks!